src/components/WeatherSuggestions/Suggestions.jsx

```diff
@@ -30,21 +29,19 @@ export default function WeatherSuggestions(results) {
         showStatus={false}
         className="carousel"
       >
-        {/* Traverse through the items map */}
-        {Object.keys(items).length > 0 &&
-          Object.keys(items).map((item) => (
-            <div key={item} className="required-item">
-              <LottieControl animationData={items[item]} />
-            </div>
-          ))}
+        {items.animations.map((item, idx) => (
```

Comment:
So we might have another problem here:

![image](https://user-images.githubusercontent.com/72784348/152353579-be829af4-80bc-4f63-a825-59490e06cd0c.png)

I think there are no checks if we fail to find a match for the weather in items.js. For me the weather is drizzle so it fails to match any suggestion (and `items` remains undefined).

Reply:
We can probably use a default or not render the component on failure to match.

Reply:
Resolved with 4f38b04